apps/api/src/auth/supertokens/supertokens.service.ts

```diff
@@ -18,6 +25,18 @@ export class SupertokensService {
         Passwordless.init({
           flowType: 'USER_INPUT_CODE',
           contactMethod: 'PHONE',
+          smsDelivery: {
+            override: (originalImplementation) => ({
+              ...originalImplementation,
+              async sendSms(input) {
+                // @ts-ignore
+                send(input.phoneNumber, input.userInputCode);
+                // TODO: before sending SMS
```

Comment:
These comments can now be removed also, you can place the service function `this.smsService.smsSend` itself here. I dont see any purpose on refactoring the function into another one called send